pkg/tools/socket.go

```diff
@@ -41,8 +45,24 @@ func GetConfig() DialConfig {
 	return cfg
 }
 
+// InitConfig allows init global DialConfig, should be called before any GetConfig(), otherwise do nothing
+func InitConfig(c DialConfig) {
```

Comment:
[minor] will this used in next part of security or can we move it to `integration/utils` because it used only there?